config/checkstyle/suppressions.xml

```diff
@@ -6,4 +6,5 @@
 
 <suppressions>
     <suppress files="[\\/]generated[\\/]" checks="."/>
+    <suppress files="ERC20Contract.java" checks="."/>
```

Comment:
As its handcrafted, it shouldn't need to be added to the suppressions in Checkstyle.